src/common/definitions.h

```diff
@@ -22,6 +22,26 @@
 #define DONT_OPTIMIZE // silently ignore on Visual Studio, where this is less of a problem
 #endif
 
+// Use these macros to enable faster math.  Put them around one or more
```

Comment:
I don't think we use the double-space-after-period convention anywhere else in the code. Let's be consistent.

Reply:
3078927